### ISSUE TITLE
Saccharopine present in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9786]
**Preferred term label:**
Saccharopine present in blood

**Synonyms**

**Definition (free text, please give PubMed ID)**
Bundled into https://www.ncbi.nlm.nih.gov/medgen/C3806359

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Saccharopinuria

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

added as
Elevated circulating saccharopine concentration
HP:0034994
